eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Handler.php

```diff
@@ -683,6 +683,6 @@ protected function removeSubtree($id, $action, $original)
      */
     protected function getHash($text)
     {
-        return md5(strtolower($text));
+        return md5(mb_strtolower($text));
```

Comment:
Should we specify the `$encoding` parameter ? I know we have done it a lot for legacy stuff